xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/forms.less

```diff
@@ -97,6 +97,9 @@ select[size="0"], select[size="1"] {
   .xErrorField {
     border-color: @state-danger-border;
   }
+  .ContentMetaLabel dl {
+    margin-bottom: 1px;   // XWIKI-16344
+  }
```

Comment:
I'd rather add the styles to ``edit.less``, right after the styles for ``#editMeta`` styles. As mentioned above, you should use the existing id for the selector, no need for an additional CSS class.

Why ``1px`` and not ``3px``? Since you are inside a LESS file the value should be either 0 or based on some existing LESS variable.

Mentioning just the JIRA issue id is not very helpful. You should also write the issue title. In any case, comments are added **before** the line not at the end of the line.

Reply:
Doing it. Yes I am now using the id.
I did `1px` as it looked fine but yes I think it is a fluke so it's not a good practice. On `line 65` we can see `.xform label` has `margin-bottom: 0.3em;` so should I use `0.3em` or leave it to `0px` ? I am not sure, I am trying the margin with both values this time.
Ok got it!

Reply:
Just tested margin: `0.3em` seems a bit more, `0px` seems fine.